backend/assets/misc/settings.json

```diff
@@ -0,0 +1,29 @@
+{
+  "marks": [
+    {
+      "name": "sans vpn",
+      "value": 100,
+      "priority": 0
+    },
+    {
+      "name": "vpn1",
+      "value": 101,
+      "priority": 0.1
+    },
+    {
+      "name": "vpn2",
+      "value": 102,
+      "priority": 0.2
+    },
+    {
+      "name": "vpn3",
+      "value": 103,
+      "priority": 0.7
+    }
+  ],
+  "games": {
+    "csgo": 101,
+    "tm": 102,
+    "lol": 103
```

Comment:
I'd rather have a list here with the available mark_s_ for the game.

Reply:
:ok_hand:
If we set the marks per game to be a list, how do you want the priority to be set (equiprobable ? list with given priority (`{ "csgo": [{101:0.3, 102:0.5, 103:0.2}],...` ? or based on priority given in the mark settings ?)

Reply:
Based on the priority given in the mark settings. You assign the mark in the subset given.

Reply:
Done